app/index.html

```diff
@@ -134,7 +134,7 @@ <h2 class="title-margin-bottom">News &amp; Events</h2>
                     <img class="news-img" src="images/2017AlumniTourneyWinners.jpg" alt="Adam, Owen and Kat, with their trophy" />
 
                    <p>The evening’s banquet was held at the spacious new Ithaca Marriott Downtown on the Ithaca Commons. The formal event was held in honor of Georges Cointe who was inducted in the Cornell Fencing 1894 Society, our hall of fame. Three alumni, David Jordan ‘62, Ritchie Thomas ‘59 and Laszlo Szerenyi ‘62, presented vignettes on their experience fencing for this illustrious Coach, who served as head coach from 1935 to 1961. A commemorative trophy was then presented to a member of Georges Cointe’s family, Marilyn Abbott. Senior speeches by Victoria Wines, Abby Forth and Jake Silverman followed with each giving much heartfelt thanks for the experience of the last four years. Winning the Georges L. Cointe Award were Lubi Kiriakidi ‘17 and Michael Maurer ‘18, the Graeme Jenning Award went to Abby Forth ‘17 and Grant Ubelele ‘20 and the Academic Excellence Award went to Lubi Kiraikidi ‘17 and Owen Sconzo ‘19.</p>
-                   <p>For those who missed a chance to reunite with old friends and meet new, you are invited to our Fall Banquet Honoring the 1967 Women’s Championship Team, October 14th 2017 in New York City.</p>
+                   <p>For those who missed a chance to reunite with old friends and meet new, you are invited to our Fall Banquet Honoring the 1967 Women’s Championship Team, September 23, 2017 at the Cornell Club in New York City. <a href="http://cornelluniversity.imodules.com/s/1717/alumni/event.aspx?sid=1717&gid=2&pgid=9077&content_id=9447" target="_blank">Register here</a></p>

```

Comment:
Just for fun, would you mind adding GA click tracking to these links? If that'll be too cumbersome, let me know, and I'll do it if I want it that badly. :smile: